mmdet/apis/inference.py

```diff
@@ -185,6 +185,7 @@ def show_result_pyplot(model,
     """
     warnings.warn('"block" will be deprecated in v2.9.0,'
                   'Please use "wait_time"')
+    warnings.warn('"fig_size" will be deprecated in v2.9.0')
```

Comment:
"fig_size" will be deprecated in v2.9.0 -> "fig_size" are deprecated and takes no effect.